README.md

```diff
@@ -1,10 +1,55 @@
-Scoville - A simple little script to collect tile statistics.
+Scoville - A tool to attribute size information in MVT tiles.
 
-Scoville generates a stream of random tile coordinates according to a given distribution and then downloads the [mapbox vector tile](https://github.com/mapbox/vector-tile-spec), recording information about the tile size, latency and contents. Aggregating this data over a long period can help decide what changes to make to tile contents, and quantify the effect of changes to the tile production.
+Current scoville commands:
+
+* `info`: Prints size and item count information about an MVT tile.
+
+### Info command ###
+
+Running `scoville info --kind kind foo.mvt` on a [Nextzen](https://nextzen.org) tile might output something like:
+
+```
+...
+.water.features.bay.cmds => 14
+.water.features.bay.count => 2
+.water.features.bay.metadata => 8
+.water.features.bay.tags => 50
+.water.features.riverbank.cmds => 61962
+.water.features.riverbank.count => 169
+.water.features.riverbank.metadata => 774
+.water.features.riverbank.tags => 3419
+.water.features.water.cmds => 54333
+.water.features.water.count => 261
+.water.features.water.metadata => 1155
+.water.features.water.tags => 5770
```

Comment:
It'd be useful to know how many tags are "name" related as a specific callout as those keys generally don't compress well.

Reply:
Errr, `properties` and `values`?

Reply:
I've tried to be more consistent about the words "tags" and "properties" in 4adb558aed32454c9f45e2fb9dd966eb9537a719 and a515daf9f944b141e4a18a5c909f465ed5c26dd3. I was mixing them up before, but now "tag" should mean a Protocol Buffer field tag, and "property" should be key-value property on a `Feature`.

The layer-level `properties` is due to how MVT stores properties. Instead of being inline in the `Feature`, all property keys and values are collected and de-duplicated at the layer. So the size for properties on each feature is only the size of the integers indexing into the layer-level key and value lists.

We could probably figure out a way of attributing the size of the shared lists onto the individual features, but for the moment I just went with the simplest thing and kept to the structure of the MVT file.